### DISCUSSION TITLE
Assign red channel from one PNG as the Alpha

### DISCUSSION THREAD

**Discussion by JokerMartini** [ImageMagick/ImageMagick#3681] [Help] [answered]
I have two images and I want to use the Red Channel from the mask.png and use it as the alpha of the beauty.png, saving the composite out as a new file called final.png

I noticed the file generated does not look correct and is super small for some reason as well. I would like it to be the same size as the source files, which are both the same size.

`'C:/helpers/montage.exe convert C:/temp/teapot_beauty.png C:/temp/teapot_mask.png -alpha off -compose CopyOpacity -composite C:/temp/final.png'`

![teapot_mask](https://user-images.githubusercontent.com/1416749/118273940-897fa180-b492-11eb-80f9-ea7f8b89ba1b.png)
![teapot_beauty](https://user-images.githubusercontent.com/1416749/118273941-897fa180-b492-11eb-9f25-3e7535bbee66.png)

**Comment by snibgo** [comment]
What version of IM, on what platform?

Why are you using "montage" when you don't want to make a montage? That is why your result is small. For IM v7, use "magick" (not "magick convert"). For IM v6, use "convert".

**Reply by JokerMartini** [reply]
I can use imagemagik. I'm on windows 10

**Reply by JokerMartini** [reply]
I have no idea how to do it that is why im asking. If you have ways of doing it with imagemagik please share :)

**Comment by snibgo** [comment, marked as answer]
What version of ImageMagick do you use? I will assume v7.
```
magick teapot_beauty.png ( teapot_mask.png -channel R -separate +channel ) -alpha off -compose CopyOpacity -composite x.png
```

**Reply by JokerMartini** [reply]
thank you
